src/test/java/io/jenkins/plugins/mcp/server/extensions/DefaultMcpServerTest.java

```diff
@@ -112,10 +112,9 @@ void testMcpToolCallTriggerBuild(JenkinsRule jenkins, JenkinsMcpClientBuilder je
                 assertThat(textContent.text()).contains("true");
             });
         }
-        await().atMost(5, SECONDS).until(() -> project.getLastBuild() != null);
+        await().atMost(10, SECONDS).until(() -> project.getLastBuild() != null);
```

Comment:
not very related to the real change but tests looks to be flaky in jenkins.io ci windows node